mysite/templates/mysite/project_bid_list.html

```diff
@@ -0,0 +1,21 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+    <h1>Project Bids</h1>
+
+    {% if object_list %}
+        <ul>
+            {% for project_bid in object_list %}
+                <li>{{ project_bid.project.name }}  :
+                    <a href="{% url "project_view" project_bid.project.id %}">View Project</a>
+                    <a href="{% url "project_bid_edit" project_bid.id %}">Edit Project Bid</a>
+                    <a href="{% url "project_bid_delete" project_bid.id %}">Delete Project Bid</a>
+                </li>
+            {% endfor %}
+        </ul>
+    {% else %}
+        Sorry, there no bid on any projects. Make bids <a href="{% url "project_list" %}">All Projects</a>
+    {% endif %}
+{% endblock %}
+
```

Comment:
This page is supposed to show the bids of a specific project. When user click on a link under project panel "show bids" then we are going to show this page with only related bids. For now i think its showing all projects bind on a single page.

Reply:
This page is for the freelancers to show all the projects and bid on it.